Marlin/src/gcode/feature/leds/M150.cpp

```diff
@@ -57,6 +68,17 @@ void GcodeSuite::M150() {
       #endif
     #endif
   );
+  #if ENABLED(LED_CONTROL_MENU)
+    led_red_intensity = (parser.seen('R') ? (parser.has_value() ? parser.value_byte() : 255) : 0);
+    led_green_intensity = (parser.seen('U') ? (parser.has_value() ? parser.value_byte() : 255) : 0);
+    led_blue_intensity = (parser.seen('B') ? (parser.has_value() ? parser.value_byte() : 255) : 0);
+    #if ENABLED(RGBW_LED) || ENABLED(NEOPIXEL_LED)
+      led_white_intensity = (parser.seen('W') ? (parser.has_value() ? parser.value_byte() : 255) : 0);
+    #endif
+    #if ENABLED(NEOPIXEL_LED)
+      led_brightness = (parser.seen('P') ? (parser.has_value() ? parser.value_byte() : 255) : pixels.getBrightness());
+    #endif
+  #endif
```

Comment:
```cpp
    const uint8_t r = parser.seen('R') ? (parser.has_value() ? parser.value_byte() : 255) : 0,
                  g = parser.seen('U') ? (parser.has_value() ? parser.value_byte() : 255) : 0,
                  b = parser.seen('B') ? (parser.has_value() ? parser.value_byte() : 255) : 0
                  #if ENABLED(RGBW_LED) || ENABLED(NEOPIXEL_LED)
                    , w = parser.seen('W') ? (parser.has_value() ? parser.value_byte() : 255) : 0
                    #if ENABLED(NEOPIXEL_LED)
                      , p = parser.seen('P') ? (parser.has_value() ? parser.value_byte() : 255) : pixels.getBrightness()
                    #endif
                  #endif
                ;

  set_led_color(r, g, b
    #if ENABLED(RGBW_LED) || ENABLED(NEOPIXEL_LED)
      , w
      #if ENABLED(NEOPIXEL_LED)
        , p
      #endif
    #endif
  );

  #if ENABLED(LED_CONTROL_MENU)
    led_red_intensity = r;
    led_green_intensity = g;
    led_blue_intensity = b;
    #if ENABLED(RGBW_LED) || ENABLED(NEOPIXEL_LED)
      led_white_intensity = w;
      #if ENABLED(NEOPIXEL_LED)
        led_brightness = p;
      #endif
    #endif
  #endif
```

Reply:
In terms of naming, it is friendlier for developers using global search to make common parts match up… So instead of
```
    led_red_intensity
    led_green_intensity
    led_blue_intensity
```
Use…
```
```
    led_intensity_red
    led_intensity_green
    led_intensity_blue
```